modules/fedimint-lnv2-client/src/cli.rs

```diff
@@ -2,17 +2,31 @@ use std::{ffi, iter};
 
 use clap::Parser;
 use fedimint_api_client::api::FederationApiExt;
+use fedimint_core::core::OperationId;
 use fedimint_core::module::ApiRequestErased;
-use fedimint_core::util::SafeUrl;
+use fedimint_core::util::{NextOrPending, SafeUrl};
+use fedimint_core::Amount;
 use fedimint_lnv2_common::endpoint_constants::{
     ADD_GATEWAY_ENDPOINT, GATEWAYS_ENDPOINT, REMOVE_GATEWAY_ENDPOINT,
 };
+use lightning_invoice::Bolt11Invoice;
 use serde::Serialize;
 
-use crate::LightningClientModule;
+use crate::{LightningClientModule, ReceiveState, SendState};
 
 #[derive(Parser, Serialize)]
 enum Opts {
+    /// Pay an invoice
+    Send {
+        gateway: SafeUrl,
+        invoice: Bolt11Invoice,
+    },
+    /// Await the final state of the send operation
+    AwaitSend { operation_id: OperationId },
```

Comment:
Any way these two (AwaitSend and AwaitReceive) could be the same command?

Reply:
I do not think so as we are awaiting different possible states here.

Reply:
@joschisan If we could figure out which one is it, we could just do the right thing.